src/functions/boundaryAreaOverlap.ts

```diff
@@ -1,96 +1,110 @@
 import {
   Sketch,
-  Feature,
   GeoprocessingHandler,
   Metric,
   Polygon,
   ReportResult,
   SketchCollection,
   toNullSketch,
-  overlapFeatures,
   rekeyMetrics,
   sortMetrics,
+  NullSketchCollection,
+  NullSketch,
+  getSketchFeatures,
   getFlatGeobufFilename,
-  isInternalVectorDatasource,
-  isExternalVectorDatasource,
-  isPolygonFeatureArray,
+  clipMultiMerge,
 } from "@seasketch/geoprocessing";
-import {
-  fgbFetchAll,
-  getFeatures,
-} from "@seasketch/geoprocessing/dataproviders";
-import bbox from "@turf/bbox";
 import project from "../../project";
+import {
+  overlapArea,
+  overlapAreaGroupMetrics,
+} from "@seasketch/geoprocessing/src";
+import {
+  firstMatchingMetric,
+  getUserAttribute,
+} from "@seasketch/geoprocessing/client-core";
+import { getPrecalcMetrics } from "../../data/bin/getPrecalcMetrics";
+import { clipSketchToSubregion } from "../util/clipSketchToSubregion";
+
+interface ExtraParams {
+  /** Optional ID(s) of geographies to operate on. **/
+  geographies?: string[];
```

Comment:
I kind of broke my own rule in suggesting this parameter name huh, should probably be geographyIds.

Reply:
Hm.... that's fair haha. Let me know if you decide to switch it